source/index.txt

```diff
@@ -80,45 +80,59 @@ more information on the MongoDB Documentation project.
 Community
 ~~~~~~~~~
 
-MongoDB has an :about:`active community </community>`. You'll get a quick
-response to MongoDB questions posted to `Stack Overflow`_.
+MongoDB has an :about:`active community </community>`.
 
-.. _`Stack Overflow`: http://stackoverflow.com/questions/tagged/mongodb
+Learning MongoDB
+~~~~~~~~~~~~~~~~
+
+In addition to the documentation, there are many ways to learn to use
+MongoDB. You can:
+
+- Enroll in a free online course at `MongoDB University
+  <https://university.mongodb.com/>`_
+
+- Browse the archive of `MongoDB Presentations
+  <https://www.mongodb.com/presentations>`_
+
+- Join a local `MongoDB User Group (MUG)
+  <https://www.mongodb.org/user-groups>`_
+
+- Attend `MongoDB World
+  <http://www.mongodb.com/lp/misc/mongodb-world-interested-in-learning-more>`_
+  or an upcoming MongoDB `event <http://www.mongodb.com/events>`_ or
+  `webinar <http://www.mongodb.com/webinars>`_
+
+- Read the `MongoDB blog <http://www.mongodb.com/blog>`_ or `Planet
+  MongoDB <http://planet.mongodb.org/>`_
+
+- Download the `Architecture Guide
+  <https://www.mongodb.com/lp/whitepaper/architecture-guide>`_
+
+Getting Help
+~~~~~~~~~~~~
+
+If you're looking for help, you'll get a quick response to MongoDB
+questions posted to Stack Overflow or to our `mailing list
+<https://groups.google.com/forum/#!forum/mongodb-user>`_. `MongoDB,
+Inc. <http://www.mongodb.com>`_ also offers commercial support and
+services.
 
 Additional Resources
 ~~~~~~~~~~~~~~~~~~~~
 
-The following resources provide additional information:
-
 .. class:: toc
 
    `MongoDB, Inc. <http://www.mongodb.com/>`_
       The company behind MongoDB.
 
-   `MongoDB Events <http://www.mongodb.com/events/>`_
-      Upcoming events where you can learn more and meet members of the
-      MongoDB community.
-
-   `Planet MongoDB <http://planet.mongodb.org/>`_
-      Aggregator of popular MongoDB blogs.
-
-   `Slides and Video <http://www.mongodb.com/presentations/>`_
-      Presentations and videos from past MongoDB events.
-
-   `MongoDB Ops Manager <http://docs.opsmanager.mongodb.com/current/>`_
-      Easiest way to run MongoDB: includes Automation, Backup, and
+   `MongoDB Ops Manager <https://opsmanager.mongodb.com/>`_
+      The easiest way to run MongoDB: includes Automation, Backup, and
       Monitoring. For more information, see the `Ops Manager
-      documentation <http://docs.opsmanager.mongodb.com/current/>`_.
+      documentation <https://docs.opsmanager.mongodb.com/current/>`_.
```

Comment:
Also, there was some weird redirection going on if you https://docs.opsmanager...   (at least it did one day before release).  With your rendered version, can you paste the rendered blurb into gmail and then click on the link?